app/src/main/res/layout/activity_new_user.xml

```diff
@@ -50,14 +50,17 @@
         android:layout_height="wrap_content"
         android:ems="10"
         android:id="@+id/editText_newPhoneNum"
-        android:hint="e.g. 0118-999-881-999-119-725... 3"
+        android:hint="e.g. 780-555-1234"
         android:inputType="number" />
 
+
+
     <Button
-        android:text="Do the make me"
+        android:text="Next"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:id="@+id/button_createNewUser"
-        android:onClick="submitNewUser"/>
+        android:id="@+id/button_nextVehicle"
+        android:onClick="nextVehicle"
+        android:layout_marginTop="10dp"/>
```

Comment:
Can we make the style of these buttons match the other button style? (We've been using the same colours, for example. Check out the buttons for making a request to see what I mean.)